grpc_api/descriptor_record_store_pb.bal

```diff
@@ -0,0 +1,166 @@
+import ballerina/grpc;
```

Comment:
It seems this is the generated file from the .proto. Do we need to commit this?

Reply:
Ideally  no need to include. However seems like there is an issue with GH Action where [grpc generation](https://github.com/ballerina-guides/ballerina-in-action-samples/pull/1/files#diff-48174d999e183bc591396cec33553f802cd00b4996a2ef08654d7ee38c2e8952R31) is not picked during compiling and causing build failure. Until thats is fixed from GitHub level thought of keeping this temporary. I will create an issue fix the workflow